### ISSUE TITLE
JS Zoom Lens for Network Images: Make the Lens Bigger?

### ISSUE THREAD

**Issue by ebeshero** [frabbitry/Ulysses#15]
@frabbitry Can you give that zoom lens a larger radius? As I'm trying to read your network graphs with it, I'm kind of only able to see part of a node at a time, and we may want to be able to see more at once.

**Comment by frabbitry** [comment]
@ebeshero I gave the zoom lens a larger radius. This is done simply by modifying the css to give the class definition for the magnifying glass a larger width and height, so I changed the width and height from 100 to 200 pixels. I can now see several nodes at a time. Do you think that it should be even bigger or is this big enough? I can also change the magnifying "strength" of the glass.

**Comment by ebeshero** [comment]
@frabbitry It's better--but I'd make it even bigger. If a center node is very large (like Sandymount Strand on the Telemachiad) you really only see that node in the lens...it'd be nice if the lens would comprehend a center and maybe one-half of the circle surrounding it? (or even larger--what if you could see a whole circle cluster?)

**Comment by ebeshero** [comment]
@frabbitry Okay, that might be *too* big...but if we can nudge it a bit so in one view we can see a center node and some of the target nodes around it, I think that's what would help me in reading the network.

**Comment by frabbitry** [comment]
@ebeshero It's now bumped up to 300px x 300px. Is that better?

**Comment by ebeshero** [comment]
Yes--much! Thank you! :-) Also the lens reading glass effect is quite wonderfully suited to your literary project site. :-)
